content/livesync/how-livesync-works.textile

```diff
@@ -0,0 +1,69 @@
+---
+title: How LiveSync works
+meta_description: "LiveSync enables you to synchronize changes in your database to frontend clients at scale."
+product: livesync
+---
+
+LiveSync is a powerful realtime data synchronization feature designed to facilitate broadcasting realtime updates from backend databases to frontend clients at scale. Leveraging Pub/Sub Channels, LiveSync ensures that data updates are propagated reliably and in order to all connected clients in realtime.
+
+LiveSync can be used for realtime collaboration, including collaborative form editing, online auctions, task management applications, or simple multiplayer turn-based games, where maintaining data consistency and low latency are important. Furthermore, developers can also build highly responsive and interactive applications with the optimistic updates design pattern offered out of the box.
+
+LiveSync has three key components, 1) a Models SDK that facilitates optimistic updates for the publisher and helps merge confirmed changes from the backend with the overall frontend state 2) a Database Connector at the backend that can watch changes in your database table using the outbox pattern and broadcast those updates and 3) the default Ably Pub/Sub channels that form the mechanism behind the scenes to broadcast those updates.
```

Comment:
I think this should be an unordered list as they aren't steps and also displayed as bullets rather than within the text.

I also don't think we need to refer to them as 'Ably Pub/Sub channels' everytime and can just use 'channels'